app/views/forms/correspondence_sending_sample_form.scala.html

```diff
@@ -0,0 +1,44 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import models.request.AuthenticatedRequest
+@import controllers.{Tab, routes}
+@import models.{Case, SampleStatus}
+@import views.forms.components.RadioOption
+@import views.html.components.confirm_or_cancel_buttons
+@import views.html.forms.components.input_radiogroup
+
+@(c: Case, field: Field)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+
+@helper.form(action = routes.SampleController.updateStatus(c.reference, Some("correspondence")), 'id -> "sample_status_form", 'novalidate -> "novalidate") {
+    @helper.CSRF.formField
+    @input_radiogroup(
+        field = field,
+        options = Seq(RadioOption(SampleStatus.AWAITING.toString, "Yes"), RadioOption("", "No")),
+        id = "sample_status_form",
+        preSelect = false,
+        label = Some(messages("case.liability.sending.sample.change-return-header")),
+        labelClass = Some ("heading-large")
+    )
+    @confirm_or_cancel_buttons(
+        id = "sending_sample_form-submit",
+        confirmText = messages("case.liability.sending.sample.change-return-confirm"),
+        cancelText = messages("case.liability.sending.sample.change-return-cancel"),
+        cancelHref = routes.CaseController.sampleDetails(c.reference).withFragment(Tab.SAMPLE_TAB.name)
```

Comment:
Just to check, are these messages intended to shared with `case.liability.___` ?

Reply:
These messages are exactly the same as liabilities so to avoid duplication we have reused them.

Reply:
Sounds reasonable!